assets/tiger.yaml

```diff
@@ -0,0 +1,24 @@
+---
+- type: model
+  name: TigerBot
+  organization: TigerResearch
+  description: TigerBot is an open source multilingual multitask LLM.
+  created_date: 2023-10-19
+  url: https://arxiv.org/pdf/2312.08688.pdf
+  model_card: https://huggingface.co/TigerResearch/tigerbot-180b-base-v2
+  modality: text; text
+  analysis: Evaluated across a range of domain tasks across standard benchmarks in comparison to predecessor LLaMA 2.
+  size: 180B parameters (dense)
+  dependencies: 
+    explanation: 7, 13, and 70B sizes trained on LLaMA 2. 180B model trained on BLOOM.
+    value: [LLaMA 2, BLOOM]
+  training_emissions: unknown
+  training_time: unknown
+  training_hardware: 32 A100-40G GPUs
+  quality_control: Safety filtering performed to mitigate risk and remove toxic content.
+  access: open
+  license: Apache 2.0
```

Comment:
The paper says this about it: "TigerBot is released under Apache-2.0 license. However, since we continual pretrained from Llama-2 (7, 13, 70B) and BLOOM (180B), one shall consult with their open-source licenses respectively." I'll add a bit about this into a license explanation.